app/services/live.js

```diff
@@ -297,6 +300,20 @@ export default class LiveService extends Service {
 
   async renderScreenVideoToPeers(peers) {
     this.peers = peers;
+    const rolesToFind = [ROLES.maven, ROLES.moderator, ROLES.guest];
+    const foundPeers = {};
+
+    for (const roleName of rolesToFind) {
+      foundPeers[roleName] = peers.find((peer) => peer.roleName === roleName);
+    }
+
+    const maven = foundPeers[ROLES.maven];
+    const moderator = foundPeers[ROLES.moderator];
+    const guest = foundPeers[ROLES.guest];
+    this.mavenRole = maven?.roleName;
+    this.moderatorRole = moderator?.roleName;
+    this.guestRole = guest?.roleName;
```

Comment:
if we are assigning object/array to these variable, then why are we keeping the default values as string

Reply:
same as above

Reply:
@bhtibrewal we are using @tracked variable for adding string which is roleName here